package.json

```diff
@@ -71,21 +71,21 @@
     "@commitlint/config-conventional": "12.1.4",
     "@commitlint/travis-cli": "12.1.4",
     "@types/eslint": "7.2.13",
-    "@types/node": "14.17.3",
+    "@types/node": "15.14.0",
     "@typescript-eslint/eslint-plugin": "4.0.1",
     "ava": "3.15.0",
     "editorconfig-checker": "4.0.2",
-    "eslint": "7.28.0",
+    "eslint": "7.29.0",
     "eslint-plugin-import": "2.23.4",
     "eslint-plugin-node": "11.1.0",
-    "eslint-plugin-promise": "4.3.1",
-    "husky": "6.0.0",
+    "eslint-plugin-promise": "5.1.0",
```

Comment:
Changelog isn't helpful in figuring out what the breaking change is. But we're not using it directly, buy via upstream, so OK I guess.